sdcm/provision/common/utils.py

```diff
@@ -118,6 +118,8 @@ def configure_sshd_script():
     sed -i "s/#MaxSessions \(.*\)$/MaxSessions 1000/" /etc/ssh/sshd_config || true
     sed -i "s/#MaxStartups \(.*\)$/MaxStartups 60/" /etc/ssh/sshd_config || true
     sed -i "s/#LoginGraceTime \(.*\)$/LoginGraceTime 15s/" /etc/ssh/sshd_config || true
+    sed -i "s/#ClientAliveInterval \(.*\)$/ClientAliveInterval 60/" /etc/ssh/sshd_config || true
+    sed -i "s/#ClientAliveCountMax \(.*\)$/ClientAliveCountMax 10/" /etc/ssh/sshd_config || true
```

Comment:
This will make sshd live-ping _all_ ssh session, while my initial suggestion was to make _client_ turn on live-pings on those sessions that are considered long and prone to connection aborts. But I've no strong preference of my plan against this one. Since its SCT-only thing, likely this doesn't change much

Reply:
the client is using python-ssh(libssh), and I'm not sure it's reading the configuration as one might expect

doing it on the server side is a bit clear that it would do what's says on the cover.

if you think the client side it more relevant, I can dig further on it.

Reply:
No, let's try with server-side configuration and see how it goes